sos/plugins/vdsm.py

```diff
@@ -114,7 +114,7 @@ def setup(self):
         ])
 
         try:
-            res = self.call_ext_prog(
+            res = self.get_cmd_output_now(
```

Comment:
with `save=False`?

Reply:
Collecting the connected storage pools output would be beneficial to RH support teams